scripts/create-comment.js

```diff
@@ -0,0 +1,44 @@
+// Adapted from ember-data https://github.com/emberjs/data/blob/master/scripts/perf-tracking/create-comment.js
```

Comment:
Please add `scripts/` to this list, then it'll fix the linting issues:

https://github.com/hjdivad/ember-m3/blob/1e8f75c397228b97cd014cd367ab5688ff392ba7/.eslintrc.base.js#L21-L35

Reply:
Good call, done and fixed/linting now passes

Reply:
Can you update this link to either a perma link or a link to a tag?

Reply:
Done